src_py/apiServer/transmitter.py

```diff
@@ -147,9 +145,7 @@ def predict(self):
         globe.experiment_flow_global.syncPredicitionWithFlow()
 
         # 1 Ack for clientsPredict(), <num of sources> Acks for updateCSV():
-        globe.pendingAcks += 1 + len(globe.components.sources) 
-
-        self.updateCSV(globe.PREDICTION_STR)
+        globe.pendingAcks += 1
```

Comment:
If you can add detailed comment about this ack variable and why it's needed.

Reply:
added